_pages/coc.md

```diff
@@ -3,3 +3,42 @@ layout: page
 title: Code of Conduct
 permalink: /coc/
 ---
+
+## tl;dr
+Follow the golden rule: treat others in the way that you would want to be treated.
+If you have a conflict with another member, or in any way feel harassed, please contact an admin to discuss the situation.
+
+## General Conduct
+All devanooga members are expected to follow these guidelines in our community:
+
+- Avoid language that is unnecessarily offensive to other users
+- Come prepared to share ideas _and defend them_. We want a place that is fun and thought provoking.
+- You may see ideas that are contrary to yours and that is ok.
+- While it is ok to express contrary ideas, it is **NOT** ok to attack, defame, or otherwise harass another member. It **IS** ok (and encouraged) to criticize ideas that you disagree with.
+- Bring a positive attitude to all interactions.
+- Assume that everyone else is being light-hearted and acting in good faith unless proven otherwise.
+
+## Conflict Resolution
+If you or any other member is being treated in a way that violates this CoC, please contact an administrator immediately. You can contact us via slack, the [contact form](/contact/) on this site, or by [email](mailto:admin@devanooga.com).
+The administrators will work to resolve the conflict in an open way unless privacy concerns prevent an open dialogue about the problem.
+
+## Conduct of Administrators
+devanooga is ran by Chattanooga-area members. We want to foster a community that is driven and shaped by the members. With that in mind:
+
+- Administrators never act alone. Every action taken by an administrator is discussed with the larger group of admins before the action is taken.
+- Administrators will do everything within reason to act transparently. In some cases, it may be necessary (for the privacy of members) to do things "behind closed doors". We will avoid this whenever possible.
```

Comment:
I would add something to the last sentence like:
>We will avoid this whenever possible, and will inform the rest of the community when such private deliberation has taken place.

The point being to keep admins accountable. If admins _must_ discuss an issue and decide on it privately, they also _must_ inform the public that such private deliberation took place. Does that make sense?